healthcheck.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+import re
+import sys
+import uuid
+from http import HTTPStatus
+from pathlib import Path
+import datetime
+import requests
+
+# Read Docker secrets
+with open("/run/secrets/client_secret") as f:
+    client_secret = f.read().strip()
+
+with open("/run/secrets/password") as f:
+    password = f.read().strip()
+
+client_id = "local_candig"
+username = "user2"
+
+def get_token(username=None, password=None, client_id=None, client_secret=None):
+    payload = {
+        "client_id": client_id,
+        "client_secret": client_secret,
+        "grant_type": "password",
+        "username": username,
+        "password": password,
+        "scope": "openid",
+    }
+    response = requests.post(
+        "http://candig.docker.internal:8080/auth/realms/candig/protocol/openid-connect/token",
```

Comment:
This needs to use the `OPA_URL` environment variable, or editing the domain name will cause the health check to never report healthy

Reply:
Updated.